src/react/databrowser/objects/MetadataSearch.jsx

```diff
@@ -69,13 +73,19 @@ const MetadataSearch = ({
       return;
     }
     dispatch(newSearchListing(bucketName, inputText));
```

Comment:
I think this line may be dropped ?

Reply:
🤔  I don't think we should drop this line, it is the action to do the search objects.